### DISCUSSION TITLE
I am dumb, but is there hope? (bricked PC)

### DISCUSSION THREAD

**Discussion by Cynthianon** [DavidS95/Smokeless_UMAF#13] [Q&A] [answered]
So I'm really dumb and I don't know how to read. I thought the « Know setting that will make your device brick » disclaimer for the Pstates settings was about if I totally exceed the device capabilities, but no, it's literally if you change even 1mV (which I did...)

So.... Is my laptop recoverable? There aren't any cmos battery and power draining it didn't work either. I located the CMOS chip, it's a mx77u25650f from Macronix, but I have absolutely no way of clearing it or resetting it...
My laptop just doesn't boot anymore. Black screen (backlight is off), fan spin, a few minutes later is power cycles. Keyboard is unresponsive (no caps lock or num lock lights, no brightness change for the backlight), plugging external display doesn't work either.

I know I deserve it for being such a fool but if there's a way to recover it I really thank you for helping me. My laptop is under warranty but I doubt they'll cover that (I'll call them tomorrow because today it's Sunday where I live).

**Comment by DavidS95** [comment, marked as answer]
Its recoverable if you manage to have a way to reflash the bios. Either through an oem designated way, for example HP has one where u hold down certain keys while on start up with a usb plugged in. Or by using a manual flashing device which requires you to by that and flash the bios chip yourself. Either way would work as long as you can get a bios entirely reflashed. Yea for some reason this setting does not get cleared when bios does normally thus needing a new bios to then be cleared.

**Comment by w568w** [comment]
For anybody comes here desperately, I have written [a guide (Chinese only)](https://www.w568w.eu.org/how-to-fix-bricked-lenovo-laptop.html) on this. :)

If you don't understand Chinese, try a translation tool. Confirmed that the content is readable with Google translation.
